sdk/keyvault/keyvault-admin/review/keyvault-admin.api.md

```diff
@@ -166,6 +170,21 @@ export interface KeyVaultSelectiveKeyRestoreResult {
     startTime: Date;
 }
 
+// @public
+export interface KeyVaultSetting {
+    name: string;
+    value: boolean;
```

Comment:
can we get a better name than `value`? If it's always going to be a boolean maybe `enabled` or some such?

Reply:
Other kinds of settings are expected at some point. Initially I had a `type` property as well to discriminate, but apparently the other languages aren't exposing that immediately. I'm thinking about adding it back.

Reply:
.NET exposes a type: https://github.com/Azure/azure-sdk-for-net/blob/c1e35999e54aa018dc5e1ba9718ac685b9524c3d/sdk/keyvault/Azure.Security.KeyVault.Administration/src/Generated/Models/KeyVaultSetting.cs#L43